packages/collabswarm/src/json-serializer.test.ts

```diff
@@ -0,0 +1,29 @@
+import { expect, test } from '@jest/globals';
+import { JSONSerializer } from './json-serializer';
+
+const json_serializer = new JSONSerializer<any, any>();
+
+let test_object = { "key": "val" };
+let test_object_serialized = '{"key":"val"}';
+let test_string = "Hello";
+let test_string_as_u8_array = Uint8Array.from([72,101,108,108,111]);
+
+test('serialize json object to string', () =>{
```

Comment:
I thought these were setup as Table-driven tests?

Reply:
That's what I've been doing since you mentioned it; these were created before that. You may be thinking of the tests I was working on when the video call started, which is the subtlecrypo package